fieldservice/models/fsm_person.py

```diff
@@ -22,7 +22,6 @@ class FSMPerson(models.Model):
     stage_id = fields.Many2one(
         "fsm.stage",
         string="Stage",
-        track_visibility="onchange",
```

Comment:
tracking=True ?

Reply:
Since v13, it's tracking instead of track_visibility 

https://github.com/odoo/odoo/commit/b8197ff1784094a21f3df077e55bcdd23df157eb

Reply:
Exactly, but you didn't add tracking=True after deleting track_visibility.  Is this intentional to remove all tracking here?

Reply:
Sorry. I remember now. :older_man: 
It's because it causes the following warning:

`WARNING db odoo.fields: Field fsm.person.stage_id: unknown parameter 'tracking', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it `

Reply:
I wonder why it works to track stage_id on fsm_order?

Either way, it isn't crucial for me to have this, so I'm OK with removing to fix the warning